src/utils/apiTestSteps.ts

```diff
@@ -0,0 +1,108 @@
+import axios from "axios";
+const API_INSTANCE = axios.create({ baseURL:"https://unb2uhs7rl.execute-api.eu-west-2.amazonaws.com/dev" });
+
+export async function stubStartPost(body:any):Promise<any> {
+	try {
+
+		const path = "https://erveje5km8.execute-api.eu-west-2.amazonaws.com/dev/start";
```

Comment:
Could we parameterise these somehow? As it stands we wont be able to reuse this test suite on multiple environments.

Reply:
Request URL's have been added as environment variables, .env.example file added to repository and repo secrets added for use in GitHub workflows